webxrlayers-1.bs

```diff
@@ -1664,7 +1664,8 @@ This module replaces the steps given by "[=update the pending layers state=]" fr
 <dfn dfn-for="layers">update the pending layers state</dfn> with {{XRSession}} |session| and {{XRRenderStateInit}} |newState|, it must run the following steps:
   1. If |newState|'s {{XRRenderStateInit/baseLayer}} is set:
     1. If the |session| was created with "[=feature descriptor/layers=]" enabled, throw an {{NotSupportedError}} and abort these steps.
-    1. If |newState|'s {{XRRenderStateInit/layers}} is not empty, throw a {{TypeError}} and abort these steps.
+    1. If |newState|'s {{XRRenderStateInit/layers}} is not <code>null</code>, throw a {{NotSupportedError}}.
```

Comment:
This matches the WebXR spec https://immersive-web.github.io/webxr/#update-the-pending-layers-state